core/src/main/resources/openapi.yaml

```diff
@@ -5376,8 +5246,27 @@
           },
           "topicDeletable" : {
             "type" : "boolean"
+          },
+          "totalNoPages" : {
+            "type" : "string"
+          },
+          "currentPage" : {
+            "type" : "string"
+          },
+          "allPageNos" : {
+            "type" : "array",
+            "items" : {
+              "type" : "string"
+            }
+          },
+          "documentation" : {
```

Comment:
We talked about this before, but this is a specific example: I think it will happen elsewhere as well, but here, because `documentation` is not `required`, the type generated is `string | undefined`. However, it will be `null` in the actual response.


![Screenshot 2023-03-15 at 09 35 56](https://user-images.githubusercontent.com/20607294/225252647-c6161829-56f0-4acd-9e4e-8400d13cb042.png)

It would be awesome if someone the openapi spec could be generated with the `nullanle` flag where appropriate.


<img width="1108" alt="Screenshot 2023-03-15 at 09 40 43" src="https://user-images.githubusercontent.com/20607294/225253903-38715734-2a7c-4e2d-85dc-317c4a634382.png">

Reply:
Actually, the best would be to have `documentation` as `required`, and make it `nullable`.
![Screenshot 2023-03-15 at 09 45 39](https://user-images.githubusercontent.com/20607294/225255080-89383d0a-cd2b-4676-a664-7078a8f216ec.png)

Then we have the exact type for the property: it is always present in the response, but it may be `string` or `null`

![Screenshot 2023-03-15 at 09 46 06](https://user-images.githubusercontent.com/20607294/225255245-af9b441c-2354-4f32-b2d5-2088da8d947f.png)

Reply:
https://www.openapis.org/blog/2021/02/16/migrating-from-openapi-3-0-to-3-1-0 nullable seems to be removed

Reply:
Yes, they recommend to define the type as follow now ^^ 

```
type:
- "string"
- "null" 
```